pyproject.toml

```diff
@@ -41,13 +41,10 @@ dependencies = [
 ]
 
 [project.urls]
-"Homepage" = "https://www.fatiando.org/harmonica"
```

Comment:
This should be the link of the documentation

Reply:
Thanks for the comment @VascoSch92. In fatiando/choclo#77 we agreed that it's redundant to have Homepage and Documentation urls pointing to the same website. This is why I'm removing Homepage and leaving Documentation only.

I think it's nice to have Homepage and Documentation in case a project has its docs and website in different urls, like xarray (https://pypi.org/project/xarray). But for Harmonica it might be redundant since its website are its docs.

Do you agree?

Reply:
Ah ok ;)

Thanks for the clarification